Update README.md to inform about the wabt dependencie

The README lists fasm nodejs and pytest as the only build/test dependencies, even though wabt is also required. Running tests without wabt installed will throw the following error

```
E       AssertionError: assert 'Please insta...ecifically.\n' == '69\n42\n69\n...7)\n1\n0\n1\n'
E         + Please install wabt and wat2wasm specifically.
E         - 69
E         - 42
E         - 69
E         - 42
E         - 0
E         - 3...
E
E         ...Full output truncated (13 lines hidden), use '-vv' to show
```

--- a/README.md
+++ b/README.md
@@ -18,8 +18,8 @@ $ python3 -V
 $ git clone https://github.com/farkon00/cont.git
 $ cd cont
 
-# Install fasm with your package manager, nodejs for testing wasm
-$ sudo apt install fasm nodejs 
+# Install fasm with your package manager, nodejs and wabt (https://github.com/WebAssembly/wabt) for testing wasm
+$ sudo apt install fasm nodejs wabt
 $ python3 -m pip install pytest
 $ pytest test.py
 

